docs(split): expands design guidelines for split layout. (#3674)
patternfly/patternfly-org: packages/documentation-site/patternfly-docs/content/design-guidelines/layouts/split.md

id: Split
section: layouts
---
The **split** layout positions items horizontally in a container, with one item filling the remaining horizontal space as the viewport is resized.

##  Elements

<img src="./img/split-example.png" alt="Example image showing an example of a split layout." width="3330px"/>

1. **Split**: The split container, which holds all split items.
1. **Split item**: An individual section of content to be placed in the split layout.

## Usage
Use the split layout when content should be flexible and may change dynamically. 

## Variations

### Multiple items that fill space 

> You can specify which split item fills the remaining horizontal space in the layout. More than one item can be set to fill, as, and they will share the remaining space equally.

Reviewer: I don't understand this sentence haha the "can be set to fill, as, and"

> You can specify which split item fills the remaining horizontal space in the layout. More than one item can be set to fill, as, and they will share the remaining space equally.

Author: ```suggestion
You can specify which split item fills the remaining horizontal space in the layout. More than one item can be set to fill, and they will share the remaining space equally.
```

id: Split
section: layouts
---
The **split** layout positions items horizontally in a container, with one item filling the remaining horizontal space as the viewport is resized.

##  Elements

<img src="./img/split-example.png" alt="PatternFly website screenshot showing modal created with bullseye layout." width="3330px"/>

1. **Split**: The split container, which holds all split items.
1. **Split item**: An individual section of content to be placed in the split layout.

## Usage
Use the split layout when content should be flexible and may change dynamically. 

## Variations

### Multiple items that fill space 

> You can specify which split item fills the remaining container space. More than one item can be set to fill, and they will share the remaining space equally.

Author: ```suggestion
You can specify which split item fills the remaining container space. More than one item can be set to fill and they will share the remaining space equally.
```

<img src="./img/split-filled.png" alt="PatternFly website screenshot showing modal created with bullseye layout." width="3330px"/>

### Items that wrap 

> You can wrap a split item to ensure that it fits among other content in the viewport as it is resized.

Author: ```suggestion
You can wrap a split item to ensure that it properly fits alongside other content as the screen is resized.
```
